server/repository/src/migrations/v1_06_00/master_list_line_add_item_link_id.rs

```diff
@@ -15,12 +15,15 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
             FOREIGN KEY (item_link_id) REFERENCES item_link(id);
             DROP INDEX IF EXISTS index_master_list_line_item_id_fkey;
             ALTER TABLE master_list_line DROP item_id;
-            "#,
+
+            CREATE INDEX "index_master_list_line_item_link_id_fkey" on "master_list_line" (item_link_id);
+        "#,
     )?;
     #[cfg(not(feature = "postgres"))]
     sql!(
         connection,
         r#"
+            DROP INDEX IF EXISTS index_master_list_line_item_id_fkey;
```

Comment:
Think its petter to remove IF EXISTS then we catch if there is a spelling mistake in the migration...

Reply:
ooooooh good call!

Reply:
Did it for dropping triggers in sqlite as well - the pgsql had different CREATE OR REPLACE and I couldn't be bothered refactoring to drops haha. sqlite should pick it up.

This change did pick up a few indexes we were dropping that never existed!